README.md

```diff
@@ -180,7 +187,19 @@ If you want to see steps then you can use `--steps` option of codeception. Check
 
 **Note**:You can modify the timeout time by setting the value of **TIMEOUT** constant lower for fast machines and higher for slow computers. The constant located in the file `tests/codeception/acceptance/_bootstrap.php`
 
-Changing the browser the tests are running with?
+#### You can run an install before individual run `feature` using following command.
+
+**_Linux & Mac_**
+```bash
+$ tests/codeception/vendor/bin/robo run:install-and-test
+```
+
+**_Windows_**
+```cmd
+$ tests\codeception\vendor\bin\robo run:install-and-test
```

Comment:
Hmm is the command not a bit long? :) How about we use run:test and you can add --install as parameter?